### DISCUSSION TITLE
Wrong result of MulTransposed on 4.5.1 (nuget) and 4.5.2 (built from source) ?

### DISCUSSION THREAD

**Discussion by jbardan** [emgucv/emgucv#529] [Q&A]
Dear community,

I have very strange behavior of CvInvoke.MulTransposed

Platform: Ubuntu 20.04
Dotnet : 5.0.6
Package: nuget 4.5.1.4349 , and local build of 4.5.2.4667

If a use CvInvoke.MulTransposed over matrix J = [1, 2], with flag set to true, and a scale set to 3 I get the result:
```
[-1, -2]
[-2, -4]
```
instead of (3xJTxJ) =
```
[3, 6]
[6, 12]

```
There are two problems with this result:
1. there is a negative sign to all expected resulting coefficients
2. scale is ignored

Do you think it is a misuse from my part ?
I've run the same code on OpenCV in C++ and there is no issue at all.

Thanks in advance for any help with this matter.

Here is the test  code
```using System;
using System.Text;
using Emgu.CV;

namespace emgucvloc
{
    class Program
    {
        static void Main(string[] args)
        {
            var testJ = new Matrix<double>(1, 2);
            testJ.Data[0, 0] = 1;
            testJ.Data[0, 1] = 2;
            var testJtJ = new Mat();
            CvInvoke.MulTransposed(testJ.Mat, testJtJ, true, null, 3);

            using (var matrix = new Matrix<double>(testJtJ.Rows, testJtJ.Cols))
            {
                testJtJ.CopyTo(matrix);
                var builder = new StringBuilder();
                for (var i = 0; i < testJtJ.Rows; i++)
                {
                    builder.Append("[");
                    for (var j = 0; j < testJtJ.Cols; j++)
                    {
                        if (j != 0) builder.Append(", ");
                            builder.Append(matrix[i,j]);
                    }
                    builder.Append("]");
                    builder.AppendLine();
                }
                Console.WriteLine(builder.ToString());
            }
        }
    }
}
```

**Comment by emgucv** [comment]
Fixed in this commit: bf999b39fcd0a0bb6a78c35772cc1493ef3c011a

Thanks for pointing that out.

**Reply by jbardan** [reply]
Great, thank you for the fix.
